fritz

```diff
@@ -172,6 +172,17 @@ def build(args):
     with open(init_file, 'wb') as f:
         f.write(''.join(out).encode('utf-8'))
 
+    # get the git log for SP and add it to the SP distro
+    p = subprocess.Popen(
+        ["git", "--git-dir=skyportal/.git", "log",
+         "--pretty=format:'%C(cyan)[%ci]%Creset %s %C(auto)%h'"],
+        stdout=subprocess.PIPE,
+        universal_newlines=True,
+    )
+    out, err = p.communicate()
```

Comment:
Since no further piping is required here, maybe use `subprocess.run`?

```suggestion
    p = subprocess.run(
        ["git", "--git-dir=.git", "log",
         "--pretty=format:'%C(cyan)[%ci]%Creset %s %C(auto)%h'"],
         capture_output=True,
         universal_newlines=True, 
         cwd="skyportal"
     )
     out = p.stdout
```